app/command.go

```diff
@@ -155,10 +156,20 @@ func (a *App) ListAllCommands(teamId string, T goi18n.TranslateFunc) ([]*model.C
 }
 
 func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *model.AppError) {
-	parts := strings.Split(args.Command, " ")
-	trigger := parts[0][1:]
+	trigger := ""
+	message := ""
+	index := strings.IndexFunc(args.Command, unicode.IsSpace)
+	if index != -1 {
+		trigger = args.Command[:index]
+		message = args.Command[index+1:]
+	} else {
+		trigger = args.Command
+	}
 	trigger = strings.ToLower(trigger)
-	message := strings.Join(parts[1:], " ")
+	if trigger[0] != '/' {
```

Comment:
Let's use `strings.HasPrefix` to make the code resilient to an empty `args.Command` which might otherwise trigger an array out of bounds error here.
```suggestion
	if !strings.HasPrefix(trigger, "/") {
```